### ISSUE TITLE
Show visible buffer number in the number format 

### ISSUE THREAD

**Issue by sandangel** [akinsho/bufferline.nvim#251]
Hi, With this mapping, is it possible to show visible buffer number in on the bufferline? Currently it only supports ordinal and buf id.
```vim
nnoremap <silent><leader>1 <Cmd>BufferLineGoToBuffer 1<CR>
nnoremap <silent><leader>2 <Cmd>BufferLineGoToBuffer 2<CR>
nnoremap <silent><leader>3 <Cmd>BufferLineGoToBuffer 3<CR>
nnoremap <silent><leader>4 <Cmd>BufferLineGoToBuffer 4<CR>
nnoremap <silent><leader>5 <Cmd>BufferLineGoToBuffer 5<CR>
nnoremap <silent><leader>6 <Cmd>BufferLineGoToBuffer 6<CR>
nnoremap <silent><leader>7 <Cmd>BufferLineGoToBuffer 7<CR>
nnoremap <silent><leader>8 <Cmd>BufferLineGoToBuffer 8<CR>
nnoremap <silent><leader>9 <Cmd>BufferLineGoToBuffer 9<CR>
```

**Comment by akinsho** [comment]
@sandangel this is essentially a duplicate of an issue opened a few days ago #249 the solution is the same. As I mentioned there I'm busy with other things at the moment so if you or the other poster of the other issue would like to contribute a PR I can try and guide a PR but I'm not likely to pick this up soon personally.

**Comment by sandangel** [comment]
hi, thanks for the heads-up. Could you please let me know your thoughts on:
- Will this be a simple task from your point of view? Like deep diving into the source code and change lots of things or just need to tweak 1, 2 functions?
- If we start implementing this, where should we look at in the code? Since I'm not familiar with the code base yet, this might help me move faster.

**Comment by akinsho** [comment]
@sandangel I've renamed the other issue and am going to close this in favour of that since it was opened earlier than this one. I'm also gonna leave some comments about a potential solution to this there as well so either you or the issue author can potentially take a look at solving this.

**Comment by akinsho** [comment]
Closed in favour of #249
